javascript/src/components/inputs/InputDropdown.jsx

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var React = require('react/addons');
+var InputsStore = require('../../stores/inputs/InputsStore');
+var Input = require('react-bootstrap').Input;
+
+var InputDropdown = React.createClass({
+    mixins: [React.addons.LinkedStateMixin],
+    PLACEHOLDER: "placeholder",
+    getInitialState() {
+        return {
+            inputs: [],
+            selectedInput: this.PLACEHOLDER
+        };
+    },
+    loadData() {
+        InputsStore.list((inputs) => {
```

Comment:
This component was actually already prepared to do that, but the inputs were not passed. Doing that now. The code is supposed to load the inputs from the store in case they are not passed to increase the reusability of the component.